src/sankeydiagram.ts

```diff
@@ -25,18 +25,22 @@
 */
 import "../style/visual.less";
 import powerbi from "powerbi-visuals-api";
-import "@babel/polyfill";
+import "core-js/stable";
```

Comment:
Move core-js initialization to constructor method like here:
https://github.com/microsoft/powerbi-visuals-gantt/blob/master/src/gantt.ts#L378-L380